src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -0,0 +1,425 @@
+---
+title: 'Advanced Data Scrubbing (Preview)'
+sidebar_order: 1
+keywords: ["pii", "gdpr", "personally identifiable data", "compliance"]
+---
+
+In addition to using [`beforeSend`]({% link _documentation/data-management/sensitive-data.md %}#custom-event-processing-in-the-sdk) in your SDK or our [server-side data scrubbing features]({% link _documentation/data-management/sensitive-data.md %}#server-side-scrubbing) to redact sensitive data, we are currently beta-testing ways to give you more granular control over what exactly our server-side data scrubbing does to your events. Additional functionality includes:
+
+* Ability to define custom regular expressions to match on sensitive data
+* More fine-grained tuning on which parts of an event to scrub
```

Comment:
Suggest "Detailed tuning of the parts of an event to scrub"

Reply:
done